input/pagecontent/use-cases.md

```diff
@@ -1,7 +1,18 @@
-## AU eRequesting R1 use cases
-This page contains the use cases for AU eRequesting for the first release of the TDG Implementation Guide (R1). This is a working page and will reflect the current state of agreement by the TDG as well as the current draft of the use case material.
+In the context of AU eRequesting, use cases focus on standirdising clinical requesting and ordering within the Australian healthcare system, specifically targeting Pathology and Imaging requests in AU eRequesting Release 1 (R1). The use cases aim to ensure that the system supports accurate and efficient data exchange and workflow management.
 
-{% include use-case-feedback.md -%}
+The use cases for AU eRequesting are divided into two main aspects:
```

Comment:
This paragraph is for a different page. There is nothing on this page that indicates workflow perspectives or payload and status transition. Remove this paragraph. Or reword so it fits this page.

Reply:
removed bullet points and reworded